bedrock/firefox/templates/firefox/new/basic/base_download.html

```diff
@@ -68,6 +68,11 @@ <h2 class="c-sub-navigation-title">{{ ftl('sub-navigation-firefox-for-desktop',
 {% block content %}
 <main role="main" class="main-download" {% if v %}data-variant="{{ v }}"{% endif %}>
 
+  {# Show unsupported notification to IE 6/7 visitors on Windows XP/Vista #}
```

Comment:
Moved this notification from the /thanks page to /new

<img width="1673" alt="image" src="https://user-images.githubusercontent.com/400117/128147628-73c0a141-52e7-4de3-89a5-41779224ba18.png">